leaflet/forms/widgets.py

```diff
@@ -25,7 +25,11 @@ def media(self):
 
         # We assume that including media for widget means there is
         # no Leaflet at all in the page.
-        js = ['leaflet/leaflet.js'] + PLUGINS[PLUGIN_FORMS]['js']
+        js = [
+            'leaflet/leaflet.js',
+            'leaflet/leaflet.extras.js',
```

Comment:
Re-adding the "extras" file here, because it's no longer coming from the form plugin dependencies.